modules/service/src/main/scala/ch/epfl/bluebrain/nexus/iam/service/Main.scala

```diff
@@ -67,6 +68,31 @@ object Main {
       val acl                  = Acls[Future](aggregate, clock)
       val downStreamAuthClient = DownstreamAuthClient(appConfig.oidc, cl, uicl)
 
+      if (appConfig.auth.adminGroups.isEmpty) {
+        logger.warning("Empty 'auth.admin-groups' found in app.conf settings")
+        logger.warning("Top-level permissions might be missing as a result")
+      } else {
+        val adminGroups = appConfig.auth.adminGroups.map(group => GroupRef(appConfig.oidc.issuer, group))
+        acl.fetch(Path./).onComplete {
+          case Success(mapping) =>
+            adminGroups.foreach {
+              adminGroup =>
+                mapping.get(adminGroup) match {
+                  case Some(permissions) if permissions(Permission.Own) =>
```

Comment:
Since trying to fetch permissions when you have `Own` permissions does not work if you don't have `Read` permissions, I would suggest we add not only `Own`, but also `Read` (and/or that we fix the requirements